docs/cookbook/log-entity-changes.md

```diff
@@ -0,0 +1,136 @@
+# Log entity changes
+
+The Shopsys platform allows you to log changes to entities.
+
+The logging system works to capture changes in the doctrine unit of work before they are flushed.
+
+After setting logging we used php attributes above class, property and method. (https://www.php.net/manual/en/language.attributes.overview.php)
+
+As an implemented sample, you can study the settings on Order, OrderItem, OrderStatus and Country entities. The principle of use will be described on these examples.
```

Comment:
It would be nice to have each sentence on a separate line. It improves the readability of a markdown file (and diffs are nicer), while it will still be rendered as a single paragraph.

I don't insist on the change in this already written article (although I would not be against it 😄 ). 
Just for the future.

Reply:
sure